release-notes-1.2.0.md

```diff
@@ -2,7 +2,7 @@
 ${version-number}
 
 #### New Features
- - [SCMOD-7821] - The converter added for standard logging format to Opensuse-tomcat-image access-log.
+ - [SCMOD-7821] - The logback converters for processId and thereadId pattern of the standard logging format is refactored to specific logging event. 
```

Comment:
- Typo "thereadId"
- "is refactored" -> "have been refactored"
- "to specific logging event" - what does this mean? Do they report as part of a specific logging event for example or do they do something else?